pycheribuild/projects/build_qemu.py

```diff
@@ -574,3 +574,53 @@ def install(self, **kwargs):
             *self.config.morello_sdk_dir.rglob("share/icons/**/qemu.bmp"),
             *self.config.morello_sdk_dir.rglob("share/icons/**/qemu.svg"),
         )
+
+
+class BuildCheriAllianceQEMU(BuildQEMUBase):
+    target = "cheri-alliance-qemu"
+    repository = GitRepository("https://github.com/CHERI-Alliance/qemu.git",
+                               default_branch="codasip-cheri-riscv_v3",
+                               force_branch=True)
+    native_install_dir = DefaultInstallDir.CHERI_ALLIANCE_SDK
+    default_targets = (
+        "riscv64-softmmu,riscv64cheri-softmmu,riscv32-softmmu,riscv32cheri-softmmu,"
+    )
+    # Turn on unaligned loads/stores by default
+    unaligned = BoolConfigOption("unaligned", show_help=False, help="Permit un-aligned loads/stores", default=False)
```

Comment:
I think we can drop this one. This was just for MIPS.

Reply:
Good point.